### PR TITLE
EVG-7846: Add builds field to Patch gql type

### PR DESCRIPTION
**Pull request by SupaJoon**
https://jira.mongodb.org/browse/EVG-7846
These schema changes will provide data for the build status icons seen on the My Patches page. 
Screen Shot 2020-04-20 at 5 07 36 PM

### CODE REVIEW

**Comment by SupaJoon** [comment]
> I ran gqlgen and there were changes. Can you double check that this is up to date?

I also noticed changes. I just pushed some code fix that.

**Comment by SupaJoon** [comment]
@john-m-liu The additional commits include the `actualMakespan` field

File: graphql/schema.graphql [hunk 1]
```diff
@@ -168,6 +168,16 @@ type Patch {
...
+  builds: [Build!]!
+}
+
+type Build {
+  id: String!
+  buildVariant: String!
+  status: String!
+  timeTaken: Duration!
+  predictedMakespan: Duration!
+  actualMakespan: Duration!
```
Comment (john-m-liu) [inline graphql/schema.graphql:180]: Either this or timeTaken would be redundant wouldn't they? Which field are you planning on using in the UI?

Comment (SupaJoon) [reply graphql/schema.graphql:180]: I will use actualMakespan and remove timeTaken

**Pull request merged**